packages/lib-react-components/src/Modal/Modal.js

```diff
@@ -1,18 +1,19 @@
 import PropTypes from 'prop-types'
 import React from 'react'
+import { Grommet } from 'grommet'
 
 import WithLayer from './WithLayer'
 import ModalBody from './components/ModalBody'
 import ModalHeading from './components/ModalHeading'
 
-function Modal ({ children, className, closeFn, pad, title }) {
+function Modal ({ children, className, closeFn, pad, theme, title }) {
```

Comment:
If you add `withTheme` to this component, you don't need to pass it in anywhere else 💯